Samples/iOS-SwiftUI/iOS-SwiftUI/UIKitScreen.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import SentrySwiftUI
+import SwiftUI
+import UIKit
+
+class CustomViewController: UIViewController {
+
+    override func loadView() {
+        super.loadView()
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let label = UILabel(frame: self.view.bounds)
+        label.text = "This is UIKit"
+        label.autoresizingMask = [.flexibleHeight, .flexibleWidth]
+        label.textAlignment = .center
+        self.view.addSubview(label)
+    }
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+    }
+}
+
+struct UIKitView: UIViewControllerRepresentable {
+    typealias UIViewControllerType = CustomViewController
+
+    func makeUIViewController(context: Context) -> CustomViewController {
+        return CustomViewController()
+    }
+
+    func updateUIViewController(_ uiViewController: CustomViewController, context: Context) {
+    }
+}
+
+struct UIKitScreen: View {
+    var body: some View {
+        SentryTracedView("UIKit in SwiftUI") {
+            UIKitView()
+        }.navigationTitle("UIKit in SwiftUI")
+    }
+}
```

Comment:
`h`: I tried this
```
struct UIKitScreen: View {
    var body: some View {
        
        SentryTracedView("UIKit in SwiftUI") {
            UIKitView()
        }.navigationTitle("UIKit in SwiftUI")
        VStack(alignment: HorizontalAlignment.center, spacing: 16) {
            Text("Some text").sentryTrace("some text")
        }
    }
}
```

And the `some text` span is a child of the UIViewController
<img width="1097" alt="Screenshot 2023-02-09 at 15 29 39" src="https://user-images.githubusercontent.com/2443292/217841309-ec685561-f9e1-4150-9d44-0276ebd5ed6b.png">

I think that could be confusing to users.

Reply:
Yep now it looks like this: <img width="795" alt="Screenshot 2023-02-13 at 15 25 20" src="https://user-images.githubusercontent.com/2443292/218484318-dcd228c0-62b7-4fbf-b462-9db50466166f.png">

https://sentry-sdks.sentry.io/discover/sentry-cocoa:d2f7f92b43ed42839ca1dba410087d83/?field=title&field=event.type&field=project&field=user.display&field=timestamp&field=replayId&homepage=true&id=18214&name=&project=5428557&query=&sort=-timestamp&statsPeriod=14d&topEvents=5&yAxis=count%28%29